src/Avalonia.Base/Point.cs

```diff
@@ -204,6 +204,18 @@ public bool Equals(Point other)
             // ReSharper enable CompareOfFloatsByEqualityOperator
         }
 
+        /// <summary>
+        /// Returns a boolean indicating whether the point is equal to the other given point
+        /// (numerically).
+        /// </summary>
+        /// <param name="other">The other point to test equality against.</param>
+        /// <returns>True if this point is equal to other; False otherwise.</returns>
+        public bool NearlyEquals(Point other)
```

Comment:
This method was already available for `Size` but not for `Point` - it's useful in unit tests if not elsewhere.